filament/src/PostProcessManager.cpp

```diff
@@ -2450,6 +2450,9 @@ FrameGraphId<FrameGraphTexture> PostProcessManager::colorGrading(FrameGraph& fg,
                             PostProcessVariant::TRANSLUCENT : PostProcessVariant::OPAQUE);
 
                 commitAndRender(out, material, variant, driver);
+
+                // Clear reference to LUT texture of color grading, in case it gets destroyed
+                mi->clearParameter("lut");
```

Comment:
Note that this may change in the future, according to the comment in `SamplerGroup::setSampler()`:

> We cannot compare two texture handles to determine if an update is needed. Texture
> handles are (quickly) recycled and therefore can't be used for that purpose. e.g. if a
> texture is destroyed, its handle could be reused quickly by another texture.
> **TODO: find a way to avoid marking dirty if the texture does not change.**
 
 If they implement better change tracking of sampler groups my current workaround may become inefficient. But I doubt it would make a measurable difference, I think the risk is theoretical only...